ui/react-components/components/Service/ServiceTypeSearch.jsx

```diff
@@ -25,15 +27,19 @@ const ServiceTypeSearch = props => {
         }
     }, [serviceUuid]);
 
+
     const getServiceType = async (serviceUuid) => {
         const service = await getService(serviceUuid);
         const serviceTypes = service.serviceTypes;
-        setServiceTypes(createDropdownOptions(serviceTypes));
+        if(serviceTypes.length === 0) {
+            setIsServiceNotSelected(true);
```

Comment:
`serviceTypes` would be enough and `isServiceNotSelected` can be removed?

Reply:
Good one.. I will try that once

Reply:
I have tried it but we are setting value for isServiceNotSelected in state and its not UseEffect.  so the intention of disabling serviceType is not met